src/k8s-engine/lima.ts

```diff
@@ -1079,9 +1254,12 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
   }
 
   async requiresRestartReasons(): Promise<Record<string, [any, any] | []>> {
-    if (this.currentAction !== Action.NONE || this.internalState === K8s.State.ERROR) {
+    if ((this.currentAction !== Action.NONE || this.internalState === K8s.State.ERROR) &&
+      (this.#actualLimaInterfaceName === this.limaInterfaceName)) {
```

Comment:
This will never be hit — if the user cancelled out of the prompt to reset, then we would be in the error state.

Reply:
That test is gone.

Reply:
obsolete code